### ISSUE TITLE
Focusable Canvas Elements

### ISSUE THREAD

**Issue by BrittanyIRL** [swissspidy/web-stories-wp#11000]
## Feature Description

All canvas elements should be focused on tab. Right now, this only works for elements without masks.

## Additional Context

Historical Context: 

PR: #5968 - https://github.com/GoogleForCreators/web-stories-wp/pull/5968
Issue: #2038

## Acceptance Criteria

- All elements focusable regardless of masking
- Karma tests reflecting the above

**Comment by BrookeGraham** [comment]
Hi @felipebochehin87 

For testing we will be using keyboard navigation and we'll also want to make sure that a screen reader announces how to enter, exit and navigate within the canvas grouping. 

1. Add one of every type of element to the canvas
2. Tab in the canvas until you reach the focus container, and outline should appear just below the editor's header (just below the title and publish buttons). You should hear a screen reader tell you to how to enter the canvas. 
3. Press Enter to "get into" the canvas layer, you should hear a screen reader tell you how to navigate while inside the canvas, as well as tell you how to exit it. 
4. Press Tab or Shift + Tab. Each time you should now be focused on a new element and it should become 'selected' in the editor. All types of elements should be focusable, including the background. 
5. Press `escape` to leave the canvas and resume being able to nav to `previous button` etc.
